extensions/git/package.json

```diff
@@ -1922,6 +1922,33 @@
           "highContrast": "#E2C08D"
         }
       },
+      {
+        "id": "gitDecoration.stageAddedResourceForeground",
```

Comment:
`stageAdded` doesn't exist. There's `untracked` and there's `added`, both states already have individual colors.